### ISSUE TITLE
RLOS in 3D ArcGIS Pro 2.1 produces empty output results

### ISSUE THREAD

**Issue by BobBooth** [Esri/visibility-addin-dotnet#250]
Testing the current Visibility Add-In with Pro 2.1, in a 3D scene, RLOS is producing empty output. Using local data UTM projected (though the Pro 3D view properties say GCS_WGS84).
![image](https://user-images.githubusercontent.com/3801392/33783438-c9013fb0-dc2a-11e7-9eb7-fb9f1fe04abd.png)

You can add points, set parameters, and run the process, but the layers that are added to the scene/TOC are empty of features.
![image](https://user-images.githubusercontent.com/3801392/33783419-b3d3b33e-dc2a-11e7-84f4-8fa499a9208c.png)

LLOS is working correctly in 3D and 2D.
RLOS is working correctly in 2D.

**Comment by BobBooth** [comment]
@ACueva @csmoore

**Comment by BobBooth** [comment]
Note in the image there are several sets of points that I'd added, tried creating RLOS for them. Results layers added each time, but all the tables are empty.

**Comment by BobBooth** [comment]
I think this was user error.
I had both the visibility add in and military tools installed.
Uninstalled both, got the latest visibility add in, and it seems to be working correctly now.

**Comment by BobBooth** [comment]
Nope. It is definitely not working correctly. It seemed to work the first  time, but subsequent attempts fail. This is an actual problem.

**Comment by lfunkhouser** [comment]
@BobBooth Please update the issue to include your environment. Also further dig into this to see what version or build this was introduced as an issue.

**Comment by BobBooth** [comment]
@lfunkhouser - issue updated. I will give it a try on an earlier version of Pro.

**Comment by BobBooth** [comment]
@lfunkhouser  The RLOS runs in Pro 1.4.0 in 3D view. I do note that I'm seeing the clipped edges of the RLOS output in this case.
![image](https://user-images.githubusercontent.com/3801392/34013655-5d784462-e0e6-11e7-8da0-86b68bdeb9e6.png)
@topowright  @csmoore

**Comment by topowright** [comment]
@BobBooth good catch that issue has been repoened in the following issue: 

https://github.com/Esri/visibility-addin-dotnet/issues/242

**Comment by BobBooth** [comment]
Xuguang said:
"...
Not sure how your mask feature class is created under the hood, but please check if the spatial references of the observer points, raster surface, and the mask feature class are all the same when the mask is created and when the visibility calculation happens.

In this case, actually a mask might not be needed. You could set a outer radius when executing the Visibility tool and achieve similar results.

Thanks,
Xuguang
..."
@csmoore - would it be possible to capture that SR data from the participating source and intermediate layers and write to a log?

**Comment by BobBooth** [comment]
I am now seeing the output being created, but the RLOS circles are being clipped.
When one point is used, both sides get clipped.
![image](https://user-images.githubusercontent.com/3801392/34743852-46905712-f559-11e7-9b98-d7745b64ca67.png)

When multiple points are used, it looks as though the two outermost get clipped.
![image](https://user-images.githubusercontent.com/3801392/34743839-3c23f194-f559-11e7-8171-1b14397e8457.png)

Coordinate system info:
![image](https://user-images.githubusercontent.com/3801392/34743871-52b7eb72-f559-11e7-9e29-0e37cb9c7363.png)

Using  a mosaic dataset in Web Mercator for base elevation data.
![image](https://user-images.githubusercontent.com/3801392/34743926-840bedcc-f559-11e7-9b5c-faa4d3405444.png)

**Comment by BobBooth** [comment]
Maybe we should detect Web Mercator and other projected but not suitable for analysis coordinate systems and warn the user?
